server/lib/activities.js

```diff
@@ -181,15 +178,14 @@ export default {
 
     // get subscription data
     if (activity.data.subscription) {
-      ({ interval } = activity.data.subscription);
+      const { interval } = activity.data.subscription;
       recurringAmount = amount + (interval ? `/${interval}` : '');
     }
 
     // get transaction data
     if (activity.data.transaction) {
       ({ amount } = activity.data.transaction);
-      ({ interval } = activity.data.transaction);
-      recurringAmount = amount + (interval ? `/${interval}` : '');
+      recurringAmount = amount;
```

Comment:
Can you add `interval` line back in? Otherwise, we won't be able to tell on Slack whether it's a one-time or a recurring donation.